src/reference/asciidoc/transformer.adoc

```diff
@@ -453,6 +453,30 @@ Starting with version 5.2, the `JsonToObjectTransformer` can be configured with
 Also this component now consults request message headers first for the presence of the `JsonHeaders.RESOLVABLE_TYPE` or `JsonHeaders.TYPE_ID` and falls back to the configured type otherwise.
 The `ObjectToJsonTransformer` now also populates a `JsonHeaders.RESOLVABLE_TYPE` header based on the request message payload for any possible downstream scenarios.
 
+[[Avro-transformers]]
+===== Apache Avro Transformers
+
+Version 5.2 added simple transformers to transform to/from Apache Avro.
+
+They are unsophisticated in that there is no schema registry; the transformers simply use the schema embedded in the `SpecificRecord` implementation generated from the Avro schema.
+
+Messages sent to the `SimpleToAvroTransformer` must have a payload that implements `SpecificRecord`; the transformer can handle multiple types.
```

Comment:
Can't this `SimpleToAvroTransformer` populate a serialized type into the headers?